packages/editor/src/components/provider/index.js

```diff
@@ -50,7 +50,25 @@ const fetchLinkSuggestions = async ( search, { perPage = 20 } = {} ) => {
 		} ),
 	} );
 
-	return map( posts, ( post ) => ( {
+	const categories = await apiFetch( {
+		path: addQueryArgs( '/wp/v2/search', {
+			search,
+			per_page: perPage,
+			type: 'category',
+		} ),
+	} );
+
+	const formats = await apiFetch( {
```

Comment:
I think the way the `awaits` are specified here will result in the API requests being made serially.

This could be a good opportunity to use `await Promise.all( ... )` to perform the requests in parallel.